src/utils.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path');
+const moment = require('moment');
+const statuses = require('./constants/statuses');
+
+const normalizeFileName = (name) => {
+  return name
+    .replace(/\s/g, '-')
+    .replace(/["']/g, '');
+};
+
+const getScreenshotPath = (testName, basePath, timestamp) => {
+  const filenamePrefix = `${normalizeFileName(testName)}_${statuses.FAILED.toLocaleUpperCase()}`;
+  const dateParts = new Date(timestamp).toString().replace(/:/g, '').split(' ');
+  dateParts.shift();
+  dateParts.pop();
```

Comment:
Please leave a comment describing what you're trying to achieve

Reply:
Fixed